docs/data/sql_functions.yml

```diff
@@ -646,6 +646,9 @@ collection:
   - sql: MAP_VALUES(map)
     table: MAP.mapValues()
     description: Returns the values of the map as array. No order guaranteed.
+  - sql: MAP_ENTRIES(map)
+    table: MAP.mapEntries()
+    description: Returns an unordered array of all entries in the given map. No order guaranteed.
```

Comment:
what means `unordered` here?

Reply:
keys are not sorted, i will remove it @snuyanzin